README.Rmd

```diff
@@ -46,7 +48,7 @@ remotes::install_github("Public-Health-Scotland/phsopendata",
 
 ### Downloading a data table with `get_resource()`
 
-To extract a specific resource, you will need it's unique identifier - resource id. This can be found in the dataset metadata, the URL of a resource's page on https://www.opendata.nhs.scot/, or extracted using `ckanr::package_show`.
+To extract a specific resource, you will need it's unique identifier - resource id. This can be found in the dataset metadata, the URL of a resource's page on https://www.opendata.nhs.scot/, or extracted using `list_resources()`.
```

Comment:
```suggestion
To extract a specific resource, you will need its unique identifier - resource id. This can be found in the dataset metadata, the URL of a resource's page on https://www.opendata.nhs.scot/, or extracted using `list_resources()`.
```